.gitignore

```diff
@@ -1,7 +1,7 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # Environment Variables
-/.env
+# /.env
```

Comment:
I prefer commit this file. As you will

Reply:
Oops, I thought the `#` commented it out. I just pushed it.